lib/hooks/useUppyUploader.js

```diff
@@ -0,0 +1,64 @@
+import { useState } from "react";
+
+import Uppy from "@uppy/core";
+import { useUppy } from "@uppy/react";
+
+import { UPPY_UPLOAD_CONFIG } from "components/Editor/MediaUploader/constants";
+import { convertToFileSize } from "components/Editor/MediaUploader/utils";
+import ActiveStorageUpload from "utils/ActiveStorageUpload";
+
+const useUppyUploader = ({
+  endpoint,
+  uppyConfig,
+  uppyUploadConfig = UPPY_UPLOAD_CONFIG,
+  onSuccess = _ => {},
+}) => {
+  const [isUploading, setIsUploading] = useState(false);
+  const [error, setError] = useState("");
```

Comment:
Let's remove the `error` state completely. Directly show the error using neetoUI `Toastr`. In `LocalUploader` component, the `error` need not be rendered as a paragraph tag.